akka-actor-typed-tests/src/test/scala/akka/actor/typed/SupervisionSpec.scala

```diff
@@ -236,13 +240,55 @@ class StubbedSupervisionSpec extends WordSpec with Matchers {
   }
 }
 
-class SupervisionSpec extends TestKit with TypedAkkaSpecWithShutdown {
+class SupervisionSpec extends TestKit("SupervisionSpec", ConfigFactory.parseString(
+  """
+    akka.loggers = [akka.testkit.TestEventListener]
+  """)) with TypedAkkaSpecWithShutdown {
 
   import SupervisionSpec._
   private val nameCounter = Iterator.from(0)
   private def nextName(prefix: String = "a"): String = s"$prefix-${nameCounter.next()}"
 
-  implicit val testSettings = TestKitSettings(system)
+  // FIXME eventfilter support in typed testkit
```

Comment:
That should work since #24386 right? Maybe leave it as-is and fix in a next PR.

Reply:
Hmm, I think I rather removed it there as the existing impl. depended on typed eventbus which I also removed.